packages/.eslintrc.json

```diff
@@ -23,7 +23,18 @@
         "lodash/import-scope": ["error", "member"],
         "unused-imports/no-unused-imports-ts": "warn",
         "@typescript-eslint/await-thenable": "error",
+        "@typescript-eslint/consistent-indexed-object-style": ["off", "record"],
         "@typescript-eslint/no-for-in-array": "error",
-        "@typescript-eslint/consistent-indexed-object-style": ["off", "record"]
+        "@typescript-eslint/no-implicit-any-catch": "error",
```

Comment:
Don't add this. We will have this in TypeScript itself in 4.4. 
See https://github.com/microsoft/TypeScript/pull/41013

Reply:
https://github.com/typescript-eslint/typescript-eslint/pull/3705